thirdeye/thirdeye-frontend/app/pods/components/rootcause-tooltip/template.hbs

```diff
@@ -0,0 +1,11 @@
+<h3>Tooltip</h3>
+
+<p>{{hoverTimestamp}}</p>
+
+<div class="container">
+  {{#each urns as |urn|}}
+    <div class="row">
+      <div class="col-xs-2">{{get labels urn}}<br/>{{get values urn}}</div>
+    </div>
+  {{/each}}
+</div>
```

Comment:
nit: new line at EOF. 

fyi, we're using eslint to catch linting errors in the IDE. I'm using VScode, but i'm pretty sure intellJ also has support for it

Reply:
fixed